frontend/src/routes/groups/[groupsPage]/[group]/[groupPage]/+page.server.ts

```diff
@@ -11,17 +15,23 @@ export const load: PageServerLoad = async ({ parent, params }) => {
 	const group = params.group;
 	const page = parseInt(params.groupPage);
 
-	const usersResponse = await getUserGroup(session.user!.email!, group, page);
-	if (!usersResponse.ok) {
-		error(usersResponse.status, { message: await usersResponse.json() });
+	const membersResponse = await getUserGroup(session.user!.email!, group, page);
+	if (!membersResponse.ok) {
+		error(membersResponse.status, { message: await membersResponse.json() });
 	}
-	const users: { email: string; has_public_key: boolean }[] = await usersResponse.json();
+	const members: { email: string; has_public_key: boolean }[] = await membersResponse.json();
+
+	const notMembersResponse = await getAllUsersWhoAreNotMembers(session.user!.email!, group);
```

Comment:
where is the user email coming from? are you sure it's being retrieved from usos?

Reply:
fixed